### DISCUSSION TITLE
overlay network issue when deploy

### DISCUSSION THREAD

**Discussion by meitouren** [louislam/dockge#495] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

I tried to deploy stack to a docker swarm and error arise
traefik-net is "driver; overlay" and not attachable.

I have tried a few working stacks container running with portainer but all failed to deploy with dockge with error "PermissionDenied desc = network traefik-net not manually attachable"

I assume dockge is able to deploy on docker swarm as it has the stack folders.
![Screenshot 2024-04-23 162808](https://github.com/louislam/dockge/assets/28998799/3ae23e2e-5fae-417e-bdbe-aa6660c7f747)
It is not deploy to the correct node as instructed too.

version: "3.8"
services:
  dashy:
    # To build from source, replace 'image: lissy93/dashy' with 'build: .'
    # build: .
    image: lissy93/dashy
    container_name: Dashy
    volumes:
      - config:/config
    deploy:
      placement:
        constraints:
          - node.hostname == prod04
    networks:
      - traefik-net
    ports:
      - target: 4000
        published: 15648
        protocol: tcp
        mode: host
    environment:
      - NODE_ENV=production
    restart: unless-stopped
    healthcheck:
      test:
        - CMD
        - node
        - /app/services/healthcheck
      interval: 1m30s
      timeout: 10s
      retries: 3
      start_period: 40s
volumes:
  config: null
networks:
  traefik-net:
    external: true

Any help is appriciated.

### 📝 Error Message(s) or Log

[+] Running 8/1
 ✔ dashy 7 layers [⣿⣿⣿⣿⣿⣿⣿]      0B/0B      Pulled                                                 36.0s 
[+] Running 2/2
 ✔ Volume "dashy_config"  Created                                                                   0.0s 
 ✔ Container Dashy        Created                                                                   0.0s 
Error response from daemon: Could not attach to network traefik-net: rpc error: code = PermissionDenied desc = network traefik-net not manually attachable

### 🐻 Dockge Version

1.4.2

### 💻 Operating System and Arch

Ubuntu 22.04.4 LTS

### 🌐 Browser

Google Chrome

### 🐋 Docker Version

26.1.0

**Comment by kayakyakr** [comment]
I got around this by manually creating an attachable overlay network. Once that's done, it works just fine.
